pkg/api/validation/validation_test.go

```diff
@@ -745,6 +745,13 @@ func TestValidatePersistentVolumeClaimUpdate(t *testing.T) {
 			oldClaim:          validClaim,
 			newClaim:          validUpdateClaim,
```

Comment:
This test already effectively tests that only volumeName changes are allowed, since both PVCs are the same except that the 2nd one adds volumeName.